src/ontology/reports/component_signature-ncit.tsv

```diff
@@ -348,17 +348,17 @@
 <http://purl.obolibrary.org/obo/NCIT_C165470>
 <http://purl.obolibrary.org/obo/NCIT_C116384>
 <http://purl.obolibrary.org/obo/NCIT_C176914>
-<http://purl.obolibrary.org/obo/NCIT_C7284>
 <http://purl.obolibrary.org/obo/NCIT_C157751>
+<http://purl.obolibrary.org/obo/NCIT_C7284>
```

Comment:
Added this issue here: https://github.com/monarch-initiative/mondo-ingest/issues/66

BUt no need to deal with this as part of this here PR!

Reply:
Sounds good. Responded to that issue with my thoughts. Will not address that in this PR.